includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -1046,20 +1011,21 @@ public function ajax_add_to_cart() {
 				define( 'WOOCOMMERCE_CART', true );
 			}
 
-			$product_id = isset( $_POST['product_id'] ) ? absint( $_POST['product_id'] ) : 0;
-			$product    = wc_get_product( $product_id );
-			$qty        = ! isset( $_POST['qty'] ) ? 1 : absint( $_POST['qty'] );
+			$product_id       = isset( $_POST['product_id'] ) ? absint( $_POST['product_id'] ) : 0;
+			$product          = wc_get_product( $product_id );
+			$post_input       = filter_input_array( INPUT_POST, [ 'shipping_pending' => FILTER_VALIDATE_BOOLEAN ] );
+			$shipping_pending = $post_input['shipping_pending'];
+			$qty              = ! isset( $_POST['qty'] ) ? 1 : absint( $_POST['qty'] );
```

Comment:
Feel free to ignore this:

`qty` seems to be the only variable and posted property that is shortened. I guess we could rename it to `quantity`. It would be a tiny code readability improvement in my opinion.

We could rename the POST data passed from the front-end too.

Reply:
done 1d16e16